src/main/resources/config/application.properties-sample

```diff
@@ -0,0 +1,38 @@
+#mobile or web
+driver.environment = mobile
+
+#path to page objects
+page.package = ru.sbtqa.tag.mobilefactory.entries
+#default wait timeout in milliseconds
+page.load.timeout = 60000
+
+video.enabled = false
+video.path.dest = /tmp/video
+video.path.temp = /tmp/video
+video.highlight.enabled = false
+
+#driver or raw
+screenshot.strategy = driver
+
+#tasks to kill before test (ONLY FOR WINDOWS)
+tasks.to.kill = iexplorer,chrome
+
+#IN CASE OF MOBILE
+#parameters for mobile driver create
+appium.url = http://127.0.0.1:4723/wd/hub
+appium.device.name = Android Emulator
+appium.device.platform = 6.0
+appium.app.package = com.android.settings
+appium.app.activity = .Settings
+
+#IN CASE OF WEB
+#parameters for web driver create
+webdriver.browser.name = Chrome
+webdriver.drivers.path = src/test/resources/webdrivers/
```

Comment:
Add comment what is optional and if in case it not pointed webdriver will searched via system PATH